doc/source/whatsnew/v1.1.0.rst

```diff
@@ -291,6 +291,7 @@ Other enhancements
 - :meth:`groupby.transform` now allows ``func`` to be ``pad``, ``backfill`` and ``cumcount`` (:issue:`31269`).
 - :meth:`~pandas.io.json.read_json` now accepts `nrows` parameter. (:issue:`33916`).
 - :meth `~pandas.io.gbq.read_gbq` now allows to disable progress bar (:issue:`33360`).
+- :func:`concat` and `~DataFrame.append` now preserve extension dtypes (:issue:`33607`, :issue:`34339`).
```

Comment:
can you put this near / extend the original concat of EA issue. It might also be helpful to give cases where this works and didn't before.

Reply:
There is not yet "an original concat of EA" entry in the whatsnew, I think (that's also the reason I added the issue number here of the previous PR for Series). 
But will add something like ", for example when combining a nullable integer column with a numpy integer column".